src/pybind/mgr/dashboard/plugins/feature_toggles.py

```diff
@@ -91,6 +107,8 @@ def cmd(mgr,
                         mgr.set_module_option(
                             self.OPTION_FMT.format(feature),
                             action == Actions.ENABLE)
+                        if action == Actions.ENABLE and feature.warning:
+                            msg += [f"{Fore.YELLOW}WARNING: {feature.warning}{Style.RESET_ALL}"]
```

Comment:
Just one thing, should we put into the mgr_utils (we should move all the command-related logic to some `mgr_command.py` file), so that other modules use the same convention?

The implementation could be like:
```py
# mgr_utils.py
class WarningStr(str):
    def __str__(self):
        return f"{Fore.YELLOW}WARNING: {super().__str__()}{Style.RESET_ALL}"
```

So we should end up using this like:
```suggestion
                            msg += [WarningStr(feature.warning)]
```

Reply:
had to name it `CLIWarning()` because it looks like `Warning` is an inbuilt class and pylint complained.